week2/homework/src/Methods.js

```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+const util = require('util');
+const fs = require('fs');
+
+const readFilePromise = util.promisify(fs.readFile);
+const writeFilePromise = util.promisify(fs.writeFile);
+const appendFilePromise = util.promisify(fs.appendFile);
+
+class Methods {
+  constructor(filePath, args) {
+    this.args = args;
+    this.filePath = filePath;
+  }
+
+  async list() {
+    const todos = await this.getFileData(this.filePath);
+
+    if (!todos) {
+      return console.log(`There is no data in '${this.filePath}' file!`);
+    }
+
+    console.log(`------\nTodos:\n------`);
+    todos
+      .trim()
+      .split('\n')
+      .forEach((todo, index) => {
+        console.log(`${index + 1}. ${todo}`);
```

Comment:
love it 👍 
![image](https://user-images.githubusercontent.com/3776580/61173776-40a9e980-a598-11e9-85c0-3b3fd3c231ca.png)